ctapipe/image/showermodel.py

```diff
@@ -0,0 +1,102 @@
+import astropy.units as u
+import numpy as np
+from scipy.stats import multivariate_normal
+from scipy.spatial.transform import Rotation as R
+
+
+__all__ = [
+    "Gaussian",
+]
+
+
+class Gaussian:
+    @u.quantity_input(
+        x=u.m,
+        y=u.m,
+        phi=u.deg,
+        theta=u.deg,
+        first_interaction=u.m,
+        width=u.m,
+        length=u.m,
+    )
+    def __init__(self, n_ch, x, y, phi, theta, first_interaction, width, length):
+        """Create a 3D gaussian shower model for imaging.
+
+        Parameters
+        ----------
+        n_ch : int
+            Number of cherenkov photons in shower
+        x : u.Quantity[length]
+            x coord of shower intersection on ground
+        y : u.Quantity[length]
+            y coord of shower intersection on ground
+        phi : u.Quantity[angle]
+            azimuthal angle defining orientation of shower
+        theta : u.Quantity[angle]
+            polar angle defining orientation of shower
```

Comment:
Polar and azimuthal angle is normally the same... Does this mean azimuth and zenith? If yes, you should probably swith to call this `azimuth` and use `altitude` instead of `zenith` to be consistent with the rest of ctapipe.

Reply:
ok yeah, in spherical coords the azimuthal angle is normally the one that is in the xy-plane and the polar angle is the one that is in the xz-plane. So they are the same as `azimuth` and `zenith`. I wasn't sure about calling them the same here since it is not guaranteed that these values describe the origin of the shower if the intersection with the ground is not at the origin (0,0,0) of this frame.

Reply:
You could always just add "relative to the shower core", and then azimuth and zenith are ok.  Though be careful as azimuth may not be defined the same direction or origin as the standard Euler angle phi, which is right-handed and starts on the X-axis.